testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -36,6 +36,7 @@
     SUPPORTS_AGGREGATION_PUSHDOWN_CORRELATION(false),
     SUPPORTS_AGGREGATION_PUSHDOWN_REGRESSION(false),
     SUPPORTS_AGGREGATION_PUSHDOWN_COUNT_DISTINCT(false),
+    SUPPORTS_AGGREGATION_PUSHDOWN_WITH_VARCHAR(false),
```

Comment:
```
SUPPORTS_AGGREGATION_PUSHDOWN_WITH_VARCHAR(fallback -> fallback.test(SUPPORTS_AGGREGATION_PUSHDOWN) && falback.test(SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY))
```

& move above `// Most connectors don't support` comment, since it's not applicable to this

Reply:
Fixed.